playbooks/init/validate_aci_params.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Validate Aci deployment variables
+  hosts: oo_first_master
+  any_errors_fatal: true
+  tasks:
+  - name: Verify Aci deployment file variable is defined
+    fail:
+      msg: >
+         The variable aci_deployment_yaml_file is not defined.
```

Comment:
Can you give an example of the aci_deployment_yaml_file?
Perhaps through a README in the roles/aci?

Reply:
I checked in a README file for ACI role